### PR TITLE
Added Drupal status report command.

### PR DESCRIPTION
**Pull request by markdorison**
## Description
Added Drupal status report command.

## Motivation / Context
Allow for easily configurable Drupal status checks in test environments.

## Testing Instructions / How This Has Been Tested
TK.

### CODE REVIEW

File: robo.drupal.example.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ custom_code_paths:
   - 'web/modules/custom'
   - 'web/themes/custom'
   - 'web/sites/*/themes'
+drupal_status_report_ignore_checks:
```
Comment (adamzimmermann) [inline robo.drupal.example.yml:8]: Can we add something to the README that explains what these values should be and how to find them?

File: src/Robo/Plugin/Commands/DrupalStatusReportCommands.php [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+     * @param int $severity
+     *   The minimum severity level to show. Defaults to 1 (warning).
+     *
+     * @throws \Robo\Exception\TaskException
+     */
+    public function drupalStatusReport($siteDir = 'default', $severity = 1): void
+    {
+        $ignoreList = implode(',', Robo::config()->get('drupal_status_report_ignore_checks'));
+        $result = $this->taskExec("$this->drupalRoot/../vendor/bin/drush status-report --format=json
+            --severity=$severity --ignore=$ignoreList")
```
Comment (adamzimmermann) [inline src/Robo/Plugin/Commands/DrupalStatusReportCommands.php:50]: Didn't realize you could pass all of these flags to the status command. 🔮

File: src/Robo/Plugin/Commands/DrupalStatusReportCommands.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            ->dir("$this->drupalRoot/sites/$siteDir/")
+            ->printOutput(false)
+            ->run();
+        $drushOutput = trim($result->getOutputData());
+        $reportJson = json_decode($drushOutput);
+        if (!is_array($reportJson) || count($reportJson) > 0) {
+            $this->say($drushOutput);
```
Comment (adamzimmermann) [inline src/Robo/Plugin/Commands/DrupalStatusReportCommands.php:57]: Is there a better method than `say`?

**Review by adamzimmermann — APPROVED** [review]
A few nitpicks, but I love that we are adding tools like this.

File: src/Robo/Plugin/Commands/DrupalStatusReportCommands.php (hunk 3 above)
Comment (markdorison) [reply src/Robo/Plugin/Commands/DrupalStatusReportCommands.php:57]: Better in what way?

Comment (adamzimmermann) [reply src/Robo/Plugin/Commands/DrupalStatusReportCommands.php:57]: More semantic and we might get fancy color coding? 🤷🏼‍♂️ 

Not a big deal, just wondered if there was another method that would make it have some warning type formatting.

Comment (markdorison) [reply src/Robo/Plugin/Commands/DrupalStatusReportCommands.php:57]: Not that I am aware of. This is how we are doing it elsewhere.

**Pull request merged**